### PR TITLE
adding two new testimonials

### PR DESCRIPTION
**Pull request by hectorkambow**
femi & shuaib

Adding two new student testimonials to testimonial section

### CODE REVIEW

**Comment by hectorkambow** [comment]
@hborrelli1 

Hey Harry, 


Could yo review this for me real quick - I think its all in place and it rendered fine on my local. 

But I rather get another page of eyes before I merge it. 

Just in case - these are 2 super new testimonials that came in today. 

Figured we could add them in the testimonial section 🚀

File: data/testimonialsData.yml [hunk 1]
```diff
@@ -17,3 +17,15 @@ testimonials:
     roles: ["Contributor", "Section Intern"]
     location: New York City, New York
     profileUrl: /authors/sophia-raji/
+  - name: Femi-ige M.
+    testimonial: The EngEd program has been extremely instrumental in my technical writing career. Being able to see my total views on each article helped me convince other publications about my writing prowess.
```
Comment (hborrelli1) [inline data/testimonialsData.yml:21]: There is a typo at the end of this line.

File: data/testimonialsData.yml [hunk 2]
```diff
@@ -17,3 +17,15 @@ testimonials:
...
+    image: /authors/femi-ige-muyiwa-oladele/avatar.jpg
+    roles: ["Contributor"]
+    location: Minna, Nigeria
+    profileUrl: /authors/femi-ige-muyiwa-oladele/
+  - name: Shuaib O.
+    testimonial: Section has been a huge part of my success story. The EngEd program contributed immensely to my technical writing journey, allowing me to move from zero knowledge about technical writing to being comfortable writing technical content.
+    image: /authors/shuaib-oseni/Avatar.jpg
+    roles: ["Contributor"]
+    location: Lagos, Nigeria
+    profileUrl: /authors/shuaib-oseni/
```
Comment (hborrelli1) [inline data/testimonialsData.yml:31]: Make sure there is an empty new line at the end of this file. Our linter configuration requires that.

**Review by hborrelli1 — APPROVED** [review]
Approved once the small tweaks are fixed,

**Comment by hectorkambow** [comment]
Thank you!

**Pull request merged**